data/changelog.txt

```diff
@@ -227,43 +249,46 @@ Major optimization of field processing.
 Adds reachability cache/filter that can reject bresenham-based LOS checks in O(1) time.
 Uses field_cache in map::get_field to speedup fire check in npc::assess_danger.
 Speedup quicksave by limiting ui redraw rate
+Prevents slow processing of NPC overmap travel when `goal` is unreachable.
+Fixes the performance bug caused by `verify_vehicle_cache` recalculation.
+Adds lightweight crafting inventory to improve NPC construction time.
 
 ## Infrastructure:
 Adds basic infrastructure for diseases.
 Allow for specialization of activity parameters.
 Improved event transformations and statistic tracking.
 Jsonize Autodoc quality.
-Add tests for character BMI / kcal / weight.
+Adds tests for character BMI / kcal / weight.
 Jsonize body parts.
 Jsonize mood modifier from bodyparts.
 Allow multiple overmap terrains in start location.
 Make test failures from other platforms more easily reproducible.
 Enforce a standard header guard format.
-Add clang-tidy check to catch incorrect use of RNGs.
+Adds clang-tidy check to catch incorrect use of RNGs.
 Adds guide to pricing items to doc folder.
-Add enchantments to bionics.
-Add test for health effect on healing rate.
-Add tests for BMI effect on speed.
+Adds enchantments to bionics.
+Adds test for health effect on healing rate.
+Adds tests for BMI effect on speed.
 Update Catch2 to v2.12.0 (from 2.9.1).
 Achievements have better up-front checking to detect definition errors.
 Tests use a different config directory by default.
 Migrate ACT_PICKUP to the activity actor system.
 Integrate behavior tree implementation into monster AI.
 Move ascii arts to their own file to avoid bloat.
 Enforce that functions should not have separate x, y parameters.
```

Comment:
```suggestion
Migrates ACT_PICKUP to the activity actor system.
Integrates a behavior tree implementation into monster AI.
Moves ascii art to its own file to avoid bloat.
Enforces that functions should not have separate x, y parameters.
```
Exactly how much is the behavior tree implementation used for currently? (I'm wondering if there should be a qualifier like "preliminary" or "basic but expandable".)

Reply:
The whole implementation is there, but it needs piecework to shift individual monster behaviors from hard-code to reside in a behavior tree data structure.  IIRC there are currently two or three behaviors migrated.